debian/patches/0002-Add-buildall.sh.patch

```diff
@@ -0,0 +1,155 @@
+From: john spurling <john.spurling@reddit.com>
```

Comment:
While the origins of `buildall.sh` are the various scripts in `mcrouter/scripts/recipes`, there's enough different stuff in there (git submodules, static linking, paths, etc etc) that it would annoying to keep maintaining diffs on them. Keeping the package build stuff in a totally separate file seems way less annoying maintenance-wise. I do think this should all eventually be moved to `debian/rules`, though -- and I'll try to pair down the code duplication where possible.